src/index.css

```diff
@@ -0,0 +1,5 @@
+body {
```

Comment:
Add come common resets here, please

Reply:
Sure, done. Hope I haven't added too much and it is fine not to add reset in a separate file in this case?

Reply:
It's enough to have it in this file. What about links `text-decoration`?